lib/bloc/lnurl/lnurl_bloc.dart

```diff
@@ -64,8 +64,7 @@ class LNUrlBloc with AsyncActionsHandler {
         })
       ])
           .where((l) => l != null)
-          .map((l) => l.toLowerCase())
-          .where((l) => isLNURL(l) || isLightningAddress(l))
```

Comment:
```suggestion
          .where((l) => l != null && (isLNURL(l) || isLightningAddress(l))
```

_I'm testing multi-line code suggestion feature_

Reply:
It is missing a `)` so I'll not apply directly from here on Github but I just copied your suggestion and I'm applying it, thank you very much 🙏